source/sass/patterns/organisms/content-header.scss

```diff
@@ -125,3 +126,42 @@
     @include rem(font-size, scale(4));
   }
 }
+
+.content-header__download_link {
+  @include inset-inline($end: 7%);
+  @include inset-block($start: 24px);
+  float: inline-end;
+  position: absolute;
+
+  @include mq($from: x-wide) {
+    @include inset-inline($end: 14%);
+    @include inset-block($start: 14px);
+  }
+
+  @include mq($from: x-wide) {
+    @include inset-inline($end: 42px);
+
+    @supports (inset-inline-end: 42px) {
```

Comment:
On the eLife site the actual site only goes down to 335 before it stops due to the top navigation within Firefox and the tags also shorten and an ellipses appears.  It seems to have the grid around this still to be added and due to the icon being position absolute I think this will fix this

![image](https://user-images.githubusercontent.com/745794/58107636-25e37600-7be2-11e9-913e-607e1914fd36.png)